.github/workflows/release.yml

```diff
@@ -7,7 +7,7 @@ on:
     branches-ignore:
       - "**"
     tags:
-      - v0.*
+      - 'v*'
```

Comment:
1.x? That way we don't accidentally bump the major version unless we have a breaking change?